spec/lib/terraform/apply_command_spec.rb

```diff
@@ -15,9 +15,14 @@
 # limitations under the License.
 
 require 'terraform/apply_command'
+require 'support/terraform/command_switches_examples'
 
 RSpec.describe Terraform::ApplyCommand do
-  let(:described_instance) { described_class.new logger: logger, state: state }
+  include_context '#color'
+
+  let(:described_instance) do
+    described_class.new logger: logger, state: state
```

Comment:
That's because the described instance isn't being initialized with the required keyword.